students/1/main.js

```diff
@@ -18,10 +18,23 @@ const ex3 = () => {
     console.log(shortestString(array));
 }
 
+const ex4 = () => {
+    let array = ["this", "is", "a", "test"];
+    console.log(countLetters(array));
+}
+
 //
 // Your functions here...
 //
 
+let countLetters = (array) => {
+    let finval = 0
+    for(let i = 0; i < 4; i++){
```

Comment:
The number 4 in this case is an example of hardcoding.  This is something we do not want to do.  Try `array.length` instead.